tests/conftest.py

```diff
@@ -59,7 +62,7 @@ def pipx_temp_env_helper(pipx_shared_dir, tmp_path, monkeypatch, request):
     #   which make tests fail (e.g. on Github ansible apps exist in /usr/bin)
     monkeypatch.setenv("PATH_ORIG", str(bin_dir) + os.pathsep + os.getenv("PATH"))
     monkeypatch.setenv("PATH_TEST", str(bin_dir))
-    monkeypatch.setenv("PATH", str(bin_dir))
+    monkeypatch.setenv("PATH", str(bin_dir) + os.pathsep + str(utils_temp_dir))
```

Comment:
```suggestion
    monkeypatch.setenv("PATH", str(bin_dir / utils_temp_dir))
```

Reply:
Seems like this will make the tests fail?

Reply:
Why? But eh this ugly construct is also used by existing code so maybe it’s better to be consistent and do the refactoring in some other PR.

Reply:
The `bin_dir` is the location for apps installed by `pipx install`, so these two directories should be separated.